runtime/compiler/gfortran.vim

```diff
@@ -1,9 +1,5 @@
 " Compiler: GNU Fortran Compiler
-" Maintainer: H Xu <xuhdev@gmail.com>
```

Comment:
This line should not be removed.
Have you contacted the maintainer? If not, please contact at first.
If the maintainer doesn't respond and if you want to become a new maintainer, you should change this line to "Previous maintainer:" and add your name as the new maintainer.

Reply:
Thank you for response and guidelines. I will do as you suggested. Meanwhile I just close this pull request.